src/main/java/hillel/spring/doctors/dto/DoctorsDtoMapper.java

```diff
@@ -0,0 +1,17 @@
+package hillel.spring.doctors.dto;
+
+import hillel.spring.doctors.model.Doctor;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.ReportingPolicy;
+
+@Mapper(componentModel = "spring", unmappedTargetPolicy = ReportingPolicy.WARN)
```

Comment:
These parameters have already been configured as compiler parameters, so  they can be removed